benchexec/tablegenerator/react-table/src/utils/utils.js

```diff
@@ -194,6 +423,27 @@ const setParam = (param) => {
 
 const stringAsBoolean = (str) => str === "true";
 
+const deepEquals = (a, b) => {
```

Comment:
We already include the [deep-equal](https://github.com/substack/node-deep-equal) library, can we reuse it here?